addon/bootstrap.js

```diff
@@ -120,11 +122,13 @@ function start(webExtension) {
     if (err.message !== "This embedded extension has already been started") {
       // TODO: Should we send these errors to Sentry? #2420
     }
+    // TODO: necessary?
+    return Promise.resolve(null);
```

Comment:
This isn't necessary, the implied return undefined is enough here to swallow up the error.